### PR TITLE
Arbitrum testnet configs

### PR DESCRIPTION
**Pull request by prestwich**
based on on #727

### CODE REVIEW

File: typescript/optics-deploy/config/testnets/rinkarby.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  watchers: ['0x22B2855635154Baa41C306BcA979C8c9a077A180'],
+  recoveryManager: '0x24F6c874F56533d9a1422e85e5C7A806ED11c036',
+  optimisticSeconds: 10,
+  recoveryTimelock: 180,
+  processGas: 850_000 * 100,
+  reserveGas: 15_000 * 100,
+};
+
+export const bridgeConfig: BridgeConfig = {
+  // weth: 'TODO',
```
Comment (anna-carroll) [inline typescript/optics-deploy/config/testnets/rinkarby.ts:49]: should this be removed?

**Review by anna-carroll — APPROVED** [review]
LGTM

File: typescript/optics-deploy/config/testnets/rinkarby.ts (hunk 1 above)
Comment (prestwich) [reply typescript/optics-deploy/config/testnets/rinkarby.ts:49]: no, it's an actual TODO 😬

Comment (prestwich) [reply typescript/optics-deploy/config/testnets/rinkarby.ts:49]: but a non-essential one

**Review by anna-carroll — APPROVED** [review]
LGTM!

**Pull request merged**